src/Xamarin.Android.Build.Tasks/Tasks/CreateAndroidResourceStamp.cs

```diff
@@ -1,37 +0,0 @@
-using System;
-using System.IO;
-using Microsoft.Build.Framework;
-using Microsoft.Build.Utilities;
-
-namespace Xamarin.Android.Tasks
-{
-	public class CreateAndroidResourceStamp : Task
-	{
-		[Required]
-		public string AndroidResgenFile { get; set; }
-		[Required]
-		public string [] AndroidResourceDest { get; set; }
-		[Required]
-		public string MonoAndroidResDirIntermediate { get; set; }
-		[Required]
-		public string AndroidResgenFlagFile { get; set; }
-
-		public override bool Execute ()
-		{
-			if ((AndroidResourceDest == null || AndroidResourceDest.Length == 0) && File.Exists (AndroidResgenFile))
-				File.Delete (AndroidResgenFile);
-			Directory.CreateDirectory (MonoAndroidResDirIntermediate);
```

Comment:
@jonathanpeppers not sure why this was here.... should we worry about writing to a directory that does not exist?

Reply:
It looks like we already create this directory earlier in the build: https://github.com/xamarin/xamarin-android/blob/f1ed561187f25d87427be3a7378bce8e3a18bfaa/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L1383-L1391

This target runs right before `_UpdateAndroidResgen`:

![image](https://user-images.githubusercontent.com/840039/59106951-3ba0ad00-88fd-11e9-9029-e7c8afa29ed1.png)